ulauncher/utils/ewmh.py

```diff
@@ -272,7 +273,7 @@ def getClientList(self):
         """
         return [self._createWindow(w) for w in self._getProperty("_NET_CLIENT_LIST")]
 
-    def getClientListStacking(self):
+    def getClientListStacking(self) -> list[xobject.drawable.Window]:
```

Comment:
This was untyped before, but pyrefly inferred the type from the usage. It uses an underlying method that can return all kinds of objects, but the way it's used will only return a this type.